.github/workflows/acceptance.yml

```diff
@@ -147,11 +147,20 @@ jobs:
           with:
             node-version: ${{ steps.nvmrc.outputs.version }}
 
+        # Use `.python-version` to avoid duplication
+        - name: Get python version from `.python-verion`
+          id: python-version
+          run: echo "::set-output name=version::$(cat .python-version)"
+
         # setup python
-        - name: Set up Python 2.7
+        - name: Set up Python ${{ steps.python-verion.outputs.version }}
           uses: actions/setup-python@v1
           with:
-            python-version: 2.7
+            python-version: ${{ steps.python-version.outputs.version}}
+        # setup pip
+        - name: Install pip 
+          run: |
+            pip install --no-cache-dir "pip>=20.0.2"
```

Comment:
`make ensure-pinned-pip` would be better here, I'd hope that `ensure-venv` would just pass. If not, don't worry about it.

Reply:
rip, looks like ensure-venv fails, so GA doesn't transparently use a venv. https://github.com/getsentry/sentry/runs/644317965

That's fine for now, pip cache is sufficient. But caching the venv would be faster, would fit nicely for a future PR.